frontend/src/components/Button/Button.tsx

```diff
@@ -1,101 +1,62 @@
-import {
-  Button as ButtonMUI,
-  ButtonProps as ButtonMUIProps,
-  buttonClasses,
-} from "@mui/base";
-import { styled } from "@mui/system";
-import { PRIMARY } from "../../theme/palette";
+import React, { ButtonHTMLAttributes } from "react";
+import { twMerge } from "tailwind-merge";
+import { tv, type VariantProps } from "tailwind-variants";
 
-export type ButtonVariantType = "text" | "contained" | "outlined";
-
-export type ButtonSizeType = "small" | "standard" | "large";
-
-export interface ButtonProps extends ButtonMUIProps {
-  /**
-   * Defines the visual style of the button as either "text", "contained", or "outlined".
-   * - "text": A button with no background and a basic appearance.
-   * - "contained": A button with a solid background color, ideally used for the principal call to action on the page.
-   * - "outlined": A button with a border and no background color.
-   */
-  variant?: ButtonVariantType;
-  /**
-   * Specifies the size of the button.
-   */
-  size?: ButtonSizeType;
-  /**
-   * Defines the button contents.
-   */
-  label?: string;
-  /**
-   * Sets the button width in number of pixels.
-   */
+export interface ButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement>,
+    VariantProps<typeof buttonVariant> {
   width?: number;
-  /**
-   * Determines whether the button should occupy all available space.
-   */
-  fullWidth?: boolean;
-  /**
-   * toggleable button (only visible in "text" button varian)
-   */
+  children?: React.ReactNode;
   active?: boolean;
+  label?: string;
 }
 
 export function Button({
-  variant = "outlined",
-  size = "standard",
-  label,
   children,
+  width,
+  active,
+  className,
+  variant,
```

Comment:
We probably need a default value for variant. The `Deploy a Process` button on the InstancesPage doesn't look good because we missed the variant property.

Reply:
I'm implicitly set it to default to the "text" variant (the base style should be the bare minimum so the text variant should be put there imo), I preferred to keep it this way or else I'll have to refactor the tailwind variant thing for this one.